src/main/java/core/basesyntax/service/impl/FruitListImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.FruitList;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitListImpl implements FruitList {
```

Comment:
The class name `FruitListImpl` is not self-explanatory. Consider renaming it to reflect the service it provides, such as `FruitTransactionServiceImpl` if it's meant to handle transactions of fruits.